test/Microsoft.AspNetCore.Server.KestrelTests/MemoryPoolIteratorTests.cs

```diff
@@ -432,15 +432,15 @@ public void SkipThrowsWhenSkippingMoreBytesThanAvailableInMultipleBlocks()
         }
 
         [Theory]
-        [InlineData("CONNECT / HTTP/1.1", ' ', true, MemoryPoolIteratorExtensions.HttpConnectMethod)]
-        [InlineData("DELETE / HTTP/1.1", ' ', true, MemoryPoolIteratorExtensions.HttpDeleteMethod)]
-        [InlineData("GET / HTTP/1.1", ' ', true, MemoryPoolIteratorExtensions.HttpGetMethod)]
-        [InlineData("HEAD / HTTP/1.1", ' ', true, MemoryPoolIteratorExtensions.HttpHeadMethod)]
-        [InlineData("PATCH / HTTP/1.1", ' ', true, MemoryPoolIteratorExtensions.HttpPatchMethod)]
-        [InlineData("POST / HTTP/1.1", ' ', true, MemoryPoolIteratorExtensions.HttpPostMethod)]
-        [InlineData("PUT / HTTP/1.1", ' ', true, MemoryPoolIteratorExtensions.HttpPutMethod)]
-        [InlineData("OPTIONS / HTTP/1.1", ' ', true, MemoryPoolIteratorExtensions.HttpOptionsMethod)]
-        [InlineData("TRACE / HTTP/1.1", ' ', true, MemoryPoolIteratorExtensions.HttpTraceMethod)]
+        [InlineData("CONNECT / HTTP/1.1", ' ', true,"CONNECT")]
```

Comment:
Any reason to hardcode `"CONNECT"` rather than using `HttpMethods.Connect`, as is done in the product code?